exporter/sapmexporter/examples/signalfx-k8s.yaml

```diff
@@ -12,7 +12,7 @@ data:
       health_check:
       http_forwarder:
         egress:
-          endpoint: "https://api.${SFX_REALM}.signalfx.com"
```

Comment:
Is this correct? Does the variable get sent to the collector as `${SFX_REALM}` or is it expanded before that?

Reply:
@dashpole ping

Reply:
This shouldn't have anything to do with k8s.  K8s doesn't do any env var expansion within a configmap.